### DISCUSSION TITLE
Flow Launcher Plugin / Integration?

### DISCUSSION THREAD

**Discussion by UtopieStudio** [Rem0o/FanControl.Releases#1815] [Ideas]
Hello.
I recently installed Flow Launcher (sort of like a Mac / Ubuntu finder bar) for Windows to test out. One thing I like straight away was the plugin (optional from plugin store) to select an audio device, which on Windows has always been a pain to do quickly with hotkeys

So with Flow Launcher and the audio select plugin, you just hotkey to activate launcher (alt + x for me), type 'ad' and then select the audio device you want to switch to.

Obviously had me thinking about FanControl integration to select profiles easily, something like 'fc' which lists profiles, and select.

**Comment by Rem0o** [comment]
Could be easily done.   FanControl has the necessary interop to do so.   

Configs available are in the configuration folder, and to use one, FanControl.exe -c [configNameHere]

**Reply by UtopieStudio** [reply]
Ah. With this info I got a cude approximation working. In settings for FL: Hotkeys > Custom Query Shortcuts.

To this I 'Add' with the following:

Shortcut = fc
Expansion = > C:\Users\David\scoop\apps\fancontrol\current\FanControl.exe -c Exhaust.json

The  '>' is the built-in shortcut in FL to run commands in Windows shell from the launcher.
Is there a way to execute without UAC popping up?
